src/constants/api.tsx

```diff
@@ -2,3 +2,18 @@ export const API_KEY = 'api_key=157c2ade7d0b335008ae899a157d8967';
 export const API_URL = 'https://api.themoviedb.org/3/';
 export const API_IMG_URL = 'https://image.tmdb.org/t/p/';
 export const API_DATE_QUERY = 'primary_release_date.lte=';
+export const API_GENRE_ID = [
+    { id: 35, name: "Comedy" },
+    { id: 18, name: "Drama" },
+    { id: 10749, name: "Romance" },
+    { id: 9648, name: "Mystery" },
+    { id: 53, name: "Thriller" },
+    { id: 16, name: "Animation" },
+    { id: 28, name: "Action" },
+    { id: 12, name: "Adventure" },
+    { id: 80, name: "Crime" },
+    { id: 14, name: "Fantasy" },
+    { id: 878, name: "Science Fiction" },
+    { id: 27, name: "Horror" },
+    { id: 10751, name: "Family" }
```

Comment:
Please use Enum from @Kachyr request here for names

Reply:
we have another solution without enum, and we will use enum later in 3 sprint

Reply:
![image](https://user-images.githubusercontent.com/41084902/115025928-937da700-9eca-11eb-80a4-e49ba0035bef.png)